new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItDomainOnPV.java

```diff
@@ -0,0 +1,700 @@
+import static oracle.weblogic.kubernetes.actions.TestActions.createDockerConfigJson;
+import static oracle.weblogic.kubernetes.actions.TestActions.createDomainCustomResource;
+import static oracle.weblogic.kubernetes.actions.TestActions.createSecret;
+import static oracle.weblogic.kubernetes.actions.TestActions.createServiceAccount;
+import static oracle.weblogic.kubernetes.actions.TestActions.dockerLogin;
+import static oracle.weblogic.kubernetes.actions.TestActions.getOperatorImageName;
+import static oracle.weblogic.kubernetes.actions.TestActions.installOperator;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.domainExists;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.isHelmReleaseDeployed;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.jobCompleted;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.operatorIsReady;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.podReady;
+import static oracle.weblogic.kubernetes.assertions.TestAssertions.serviceExists;
+import static oracle.weblogic.kubernetes.extensions.LoggedTest.logger;
+import static org.awaitility.Awaitility.with;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * Tests to create domain on persistent volume using WLST and WDT.
+ */
+@DisplayName("Verify the domain can be created on persistent volume")
+@IntegrationTest
+public class ItDomainOnPV implements LoggedTest {
+
+  private static String opNamespace = null;
+  private static String domainNamespace = null;
+
+  private static String dockerConfigJson = "";
+
+  private final String domainUid = "domain-onpv";
+  private final String clusterName = "pv-domain-cluster";
+  private final String adminServerName = "admin-server";
+  private final String adminServerPodName = domainUid + "-" + adminServerName;
+  private final String managedServerNameBase = "managed-server";
+  String managedServerPodNamePrefix = domainUid + "-" + managedServerNameBase;
+
+  private final int replicaCount = 2;
+
+  private final String pvName = domainUid + "-pv"; // name of the persistent volume
+  private final String pvcName = domainUid + "-pvc"; // name of the persistent volume claim
+
+  private String wlSecretName;
+  private final String adminUser = "system";
+  private final String adminPassword = "gumby1234";
+
+  private static final ConditionFactory withStandardRetryPolicy
+      = with().pollDelay(2, SECONDS)
+          .and().with().pollInterval(10, SECONDS)
+          .atMost(5, MINUTES).await();
+
+  /**
+   * Install operator.
+   *
+   * @param namespaces injected by JUnit
+   */
+  @BeforeAll
+  public static void initAll(@Namespaces(2) List<String> namespaces) {
+    // create standard, reusable retry/backoff policy
+
```

Comment:
Is this missing or did you no longer need this?

Reply:
Still need it but moved few lines above where the condition factory is created.